qeb-hwt-github-app/overlays/stage/configmap.yaml

```diff
@@ -0,0 +1,8 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: qeb-hwt
+data:
+  subdomain: qeb-hwt
+  webhook-callback-url: "http://qeb-hwt-webhook-receiver-thoth-frontend-stage.apps.ocp.prod.psi.redhat.com"
```

Comment:
didn't we say we cannot have three apps deployed, because they would all received check runs? See: https://github.com/thoth-station/qeb-hwt/issues/93